app/views/reservations/list.html.haml

```diff
@@ -1,3 +1,8 @@
+= content_for :head_content do
```

Comment:
Technically, this doesn't need to be `= content_for`, but could be `- content_for`, but I see that this is a pattern which is pretty consistently applied to this codebase.